gui/default/syncthing/folder/editFolderModalView.html

```diff
@@ -284,6 +284,45 @@
               </p>
             </div>
           </div>
+
+          <div class="row">
+            <div class="col-md-6 form-group">
+              <p>
+                <label translate>Ownership</label>
+                &nbsp;<a href="{{docsURL('advanced/folder-sync-ownership')}}" target="_blank"><span class="fas fa-question-circle"></span>&nbsp;<span translate>Help</span></a>
+              </p>
+              <label>
+                <input type="checkbox" ng-disabled="currentFolder.type == 'sendonly' || currentFolder.type == 'receiveencrypted'" ng-model="currentFolder.syncOwnership" /> <span translate>Sync Ownership</span>
+              </label>
+              <p translate class="help-block">
+                Enables sending ownership information to other devices, and applying incoming ownership information. Typically requires running with elevated privileges.
+              </p>
+              <label>
+                <input type="checkbox" ng-disabled="currentFolder.type == 'receiveonly' || currentFolder.type == 'receiveencrypted' || currentFolder.syncOwnership" ng-model="currentFolder.sendOwnership" /> <span translate>Send Ownership</span>
```

Comment:
```suggestion
                <input type="checkbox" ng-checked="currentFolder.sendOwnership || currentFolder.syncOwnership" ng-disabled="currentFolder.type == 'receiveonly' || currentFolder.type == 'receiveencrypted' || currentFolder.syncOwnership" ng-model="currentFolder.sendOwnership" /> <span translate>Send Ownership</span>
```

Reply:
This makes the "send" checkbox disabled and ticked when "sync" is ticked. The first condition is needed when the user ticks "send" first, only then ticks "sync", and then unticks "sync" again, or otherwise "send" will automatically untick as well.

![image](https://user-images.githubusercontent.com/5626656/189383610-debb18fb-9131-4ca1-87c2-d8f12ec6e434.png)

The same applies to the extended attributes part below.